Iceberg-Pharo7.package/FTTableMorph.extension/instance/selectFirst.st

```diff
@@ -0,0 +1,3 @@
+*Iceberg-Pharo7
+selectFirst
+	self selectIndex: 1
```

Comment:
Same here

Reply:
Those methods are in a fast table in Pharo 8. I use them in this PR and I add them in a compatibility package for Pharo 7 because they are not available in Pharo 7.